queries.sql

```diff
@@ -14,4 +14,37 @@ SELECT * from animals WHERE neutered = TRUE;
 
 SELECT * from animals WHERE name != 'Gabumon';
 
-SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```

Comment:
Great work @NicolasAndrehh 🤝

- [ ] You seem to have missed the first transaction. Kindly UPDATE the animals' table by setting the species column to unspecified, then roll back the change. You may check your project requirements for more clarity.